config/schedule.rb

```diff
@@ -11,13 +11,42 @@
 # Run a daily rake task to harvest new thumbnail images.
 # These could be from new records or from records
 # where there was a previous error during harvesting.
-every :day, at: "11:00 PM", roles: [:db] do
+every :day, at: "11:00 PM", roles: [:index] do
   rake "gblsci:images:harvest_new"
 end
 
 # Run a weekly rake task to harvest thumbnail in incomplete states.
 # These could be from new records or from records
 # where there was a previous error during harvesting.
-every :saturday, at: "11:00 AM", roles: [:db] do
+every :saturday, at: "11:00 AM", roles: [:index] do
   rake "gblsci:images:harvest_retry"
 end
+
+# Harvest new records from other GeoBlacklight sites
+every "0 1 1 * *", roles: [:index] do
```

Comment:
Suggestion: Can you add some comments about what "0 1 1 * *" etc means? Maybe convert this to a more wordy format that whenever supports?

Reply:
Whenever doesn't support english translations for these, but I can add some comments.